realsense2_camera/include/t265_realsense_node.h

```diff
@@ -19,13 +20,26 @@ namespace realsense2_camera
 
         private:
             void initializeOdometryInput();
+            void setupMapReutilization();

```

Comment:
Function called at startup to setup the map saving service server, and to load a map if the input map parameter is non-empty